lib/ops/opsservice/configure.go

```diff
@@ -1187,12 +1180,28 @@ func toObject(in interface{}) (map[string]interface{}, error) {
 	return out, nil
 }
 
-func (s *site) getTeleportNodeConfig(ctx *operationContext, masterIPs []string, configPackage loc.Locator, node *ProvisionedServer) (*ops.RotatePackageResponse, error) {
-	joinToken, err := s.service.GetExpandToken(s.key)
+func (s *site) getTeleportAuthToken() (string, error) {
+	tokens, err := s.backend().GetSiteProvisioningTokens(s.domainName)
 	if err != nil {
-		return nil, trace.Wrap(err)
+		return "", trace.Wrap(err)
+	}
+	// See if there's a teleport specific token.
+	for _, t := range tokens {
+		if t.IsTeleport() && t.IsPersistent() {
+			return t.Token, nil
+		}
 	}
+	// Fallback to expand tokens for backwards compatibility.
+	for _, t := range tokens {
+		if t.IsExpand() && t.IsPersistent() {
+			return t.Token, nil
+		}
+	}
+	return "", trace.NotFound("could not find persistent teleport or expand token among %v",
```

Comment:
Is this user facing error? What could be added to help user in search for the culprit?

Reply:
It's not meant to be seen by a user. This would mean a some sort of bug or a programming error on our part as these tokens are always supposed to exist, not sure what we can add here to explain that they might be seeing this due to a bug :)